src/quo2/components/user_avatar.cljs

```diff
@@ -0,0 +1,142 @@
+                   :small (if ring?
+                            0
+                            -2)
+                   0)]
+      [rn/view {:style {:background-color (if online?
+                                            colors/success-50
+                                            colors/neutral-40)
+                        :width dimensions
+                        :height dimensions
+                        :border-width border-width
+                        :border-radius dimensions
+                        :border-color (if dark?
+                                        colors/black
+                                        colors/white)
+                        :position :absolute
+                        :bottom bottom
+                        :right right}}])))
+
+(defn container-styling [inner-dimensions outer-dimensions]
+  {:width inner-dimensions
+   :position :absolute
+   :top (/ (- outer-dimensions inner-dimensions) 2)
+   :left (/ (- outer-dimensions inner-dimensions) 2)
+   :height inner-dimensions
+   :border-radius inner-dimensions})
+
+(defn container [inner-dimensions outer-dimensions & children]
+  (let [dark-kw (if dark? :dark :light)]
+    [rn/view {:style (merge {:background-color (colors/custom-color
+                                                :turquoise
+                                                dark-kw)
+                             :justify-content :center
+                             :align-items :center}
+                            (container-styling inner-dimensions outer-dimensions))}
+     children]))
+
+(defn user-avatar
+  [{:keys [ring?
+           online?
+           size
+           status-indicator?
+           profile-picture
+           full-name]
+    :or {full-name "john doe"
+         status-indicator? true
+         online? true
+         size :big
+         ring? true}}]
+  (let [initials (if full-name
+                   (reduce str (map first (split full-name " ")))
+                   "")
+        first-initial-letter (if full-name
+                               (first full-name)
+                               "")
+        small-sizes #{:small :xs :xxs :xxxs}
+        identicon-sizes #{:big :medium :small}
+        identicon? (contains? identicon-sizes size)
+        small? (contains? small-sizes size)
+        using-profile-picture? (-> profile-picture
+                                   blank?
+                                   false?)
+        outer-dimensions (get-in sizes [size :outer])
+        inner-dimensions (get-in sizes [size (if ring?
+                                               :inner
+                                               :outer)])
+        font-size (get-in sizes [size :font-size])]
+    [rn/view {:style {:width outer-dimensions
+                      :height outer-dimensions
+                      :border-radius outer-dimensions}}
+     (when (and ring? identicon?)
+       [icons/icon :main-icons/identicon-ring32 {:width outer-dimensions
```

Comment:
you need to use icon from quo2 and size property

Reply:
Done